common/redis.go

```diff
@@ -0,0 +1,61 @@
+package common
```

Comment:
This should be moved to either `pkg/redis/redis.go` or just `storage/redis/common.go` (or something like that), because nothing else in chihaya uses redis (yet).

Reply:
Move common package to `storage/redis/common` : )